test/pool.test.ts

```diff
@@ -22,9 +22,10 @@ async function addr(idx: number): Promise<string> {
 // It may be problematic in some tests, because they will see slightly different blockchain states.
 // This function allows a `view` function to see exactly the same state as the next non-`view` one.
 async function callOnNextBlock<T>(fn: () => T): Promise<T> {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const snapshot = await buidler.ethers.provider.send("evm_snapshot", []);
   await mineBlocks(1);
-  const returned = await fn();
+  const returned = fn();
   await buidler.ethers.provider.send("evm_revert", [snapshot]);
```

Comment:
Do you mean that the linter is wrong complaining about line 26? I don’t think so. The rule disallows declarations of variables of type any. And `provider.send` returns `any` by definition. Your fix only masks this because snapshot is of type `any[]` now. I’d rather be explicit here.

Reply:
I get your point. I think it’s better to keep the ignore comment and be explicit here. Otherwise somebody might come along in the future, see the code with the array wrapping, and rewrite it more idiomatically—only to figure out that the linter complains.